### ISSUE TITLE
Add random variation to keepalives to render cyphertext harder to analyze

### ISSUE THREAD

**Issue by jennamagius** [oxy-secure/oxy#64]
Probably including sending a mixture of Ping and DummyMessage messages, in random quantities at random delays.

**Comment by myfreeweb** [comment]
Also would be cool to have a complete traffic-analysis-paranoia mode :D

- keystrokes and stuff are queued
- fixed timer loop sends messages from the queue || dummy when there's nothing on the queue

**Comment by jennamagius** [comment]
Yeah, totally inline with the the goals of the project. If you're going to batch up PtyInput for transport security reasons then you probably wind up implementing Mosh style preview-typeahead by accident.

**Comment by dcarosone** [comment]
I was looking for a place to suggest exactly this, and combining acks. Also it's good for basic bandwidth reduction, if multiple keystrokes fit in a single 272-byte frame. :-)
